### DISCUSSION TITLE
Share a code space

### DISCUSSION THREAD

**Discussion by maziac** [community/community#30947] [Codespaces]
Is there any way to share a code space with someone else.

I am not sure if I get the point for code spaces, but I thought it would be possible to setup and maintain e.g. a development environment in a Codespace that could be maintained and updated while other people use the same Codespace as their personal development environment where they can also do some personalization on top.

**Comment by craiglpeters** [comment]
The idea is that each contributor would have their own codespace working on the same shared code base in a GitHub repository. Using shared [configuration](https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/introduction-to-dev-containers) to make sure that everyone gets a great working environment, but then [personalizing](https://docs.github.com/en/codespaces/customizing-your-codespace/personalizing-github-codespaces-for-your-account) the environment to your preferences "on top" of that.

Then you use the workflow of [pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) from your branches or forks share your work.

Does this help?

**Comment by maziac** [comment]
Ok, that helps.

I'm not in an organization but in the beta.
So I guess I cannot share with "normal" GitHub users.
Is this right?
